lib/determinator/retrieve/dynaconf.rb

```diff
@@ -0,0 +1,31 @@
+require 'faraday'
+
+module Determinator
+  module Retrieve
+    # A class which loads features from Dynaconf server
+    class Dynaconf
+      # @param :host [String] The host for local Dynaconf server
+      # @param :client [String] Faraday client instance, defaults to a new instance
+      def initialize(host:, client: default_client)
```

Comment:
Non blocking:

how do you feel about calling this `url`, `uri` or `address` instead? The reason is that `host` is usually interpreted as "hostname without port", and in fact it's common to configure libraries with distinct `host` and `port` values, as a pair.

A more practical reason: when I read this I thought that only the hostname would be required, as if Dynaconf were always available on the same port (which could very well be the case!). In the initializer I saw `'localhost:2345'` however, which made me think that both could be dynamic. I've found it a bit surprising.

Another reason is that as far as I'm aware Dynaconf is supposed to always be available on localhost (i.e. on the host EC2 instance, where all containers can use it), and I would find it clearer if the config option for the hostname and the port were separate.